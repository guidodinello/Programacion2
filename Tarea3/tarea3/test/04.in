# Prueba de mayor, removerMayor, removerDeBinario
insertarEnBinario (10,10.10)
insertarEnBinario (9,9.9)
insertarEnBinario (11,11.11)
insertarEnBinario (8,8.8)
insertarEnBinario (12,12.12)
imprimirBinario
mayor
removerMayor
mayor
imprimirBinario
buscarSubarbol 10
buscarSubarbol 11
buscarSubarbol 9
removerDeBinario 9
imprimirBinario
removerMayor
imprimirBinario
removerDeBinario 10
removerDeBinario 8
imprimirBinario
Fin
