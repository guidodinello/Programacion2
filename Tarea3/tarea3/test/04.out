1>#  Prueba de mayor, removerMayor, removerDeBinario.
2>Insertado.
3>Insertado.
4>Insertado.
5>Insertado.
6>Insertado.
7>
--(12,12.12)
-(11,11.11)
(10,10.10)
-(9,9.90)
--(8,8.80)
8>(12,12.12)
9>Removido el mayor.
10>(11,11.11)
11>
-(11,11.11)
(10,10.10)
-(9,9.90)
--(8,8.80)
12>(10,10.10)
13>(11,11.11)
14>(9,9.90)
15>Removido.
16>
-(11,11.11)
(10,10.10)
-(8,8.80)
17>Removido el mayor.
18>
(10,10.10)
-(8,8.80)
19>Removido.
20>Removido.
21>
22>Fin.
